Assets/Scripts/Gamestate/AIManager.cs

```diff
@@ -160,12 +198,45 @@ private IEnumerator LookForTargets()
             DestinationTarget = player.AiTarget;
             ShootingTarget = player.AiAimSpot;
         }
+        agent.stoppingDistance = ShootingTarget ? shootingStoppingDistance : itemStoppingDistance;
         agent.SetDestination(DestinationTarget.position);
 
-        yield return new WaitForSeconds(1f);
+        yield return new WaitForSeconds(3f);
         StartCoroutine(LookForTargets());
     }
 
+    private void AnimateOffMesh()
```

Comment:
Shouldn't this be AnimateJump since it calls AnimateJumpCurve?

Reply:
Sure, seems more to the point